### PR TITLE
Change default aggregation for Counter, Observer and Recorder

### LINKED ISSUE #25: Handling timestamp appropriately based on MetricKind
In current spec and implementation of opentelemetry-go, it doesn't return timestamps in aggregated data. (ref. https://pkg.go.dev/go.opentelemetry.io/otel@v0.5.0/sdk/export/metric/aggregator)

But, for Cloud Monitoring, timestamps (`TimeInterval`) are required for [`Point`](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries#Point)

In the current implementation (#24), I leave this with tentative implementation but this should be handled in the upstream or workarounds in metric exporter.

### LINKED ISSUE #37: metric - counter example doesn't work properly
I have tried simple Counter example, it doesn't show up properly in metrics dashboard.
Both rate/delta aligners show constantly growing counter instead of rate.
I had a simple loop, incrementing counter every 100ms and it supposed to show rate of 10requests/sec, instead it seems to show something constantly growing (not a raw value though as well).

### PR DESCRIPTION
**Pull request by ymotongpoo**
* Fixes #37 
* Fixes #25 
* Change metric exporter version 0.2.0 -> 0.2.1

### CODE REVIEW

File: exporter/metric/metric.go [hunk 1]
```diff
@@ -86,6 +67,10 @@ type metricExporter struct {
 	mdCache map[key]*googlemetricpb.MetricDescriptor
 
 	client *monitoring.MetricClient
+
+	// startTime is the cache of start time shared among all CUMULATIVE values.
+	// c.f. https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors#MetricKind
+	startTime time.Time
```
[the remark this answers is not in the document]
Comment (ymotongpoo) [reply exporter/metric/metric.go:73]: Done

File: exporter/metric/selector.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/sum"
+)
+
+type selectorCloudMonitoring struct{}
+
+var _ export.AggregationSelector = selectorCloudMonitoring{}
+
+// NewWithCloudNewWithCloudMonitoringDistribution return a simple aggregation selector
+// that uses lastvalue, counter, array, and aggregator for three kinds of metric.
+// NOTE: this selector is used to
```
[the remark this answers is not in the document]
Comment (ymotongpoo) [reply exporter/metric/selector.go:31]: Sorry. I added more comments here.

**Comment by ymotongpoo** [comment]
@rghetia thank you for reviewing. done fixing.

**Comment by rghetia** [comment]
adding comment to trigger CLA.

**Comment by rghetia** [comment]
closing and opening to trigger CLA.

**Comment by james-bebbington** [comment]
> Isn't it worth to make a release with this fix? Otherwise, it seems that metrics aren't usable. Sure I can specify a commit in my `go.mod`, but it took me some time to find out the solution for the issue because I used the latest released version.

I've just made another release. Note that these exporters are still in the process of being built, so expect some breaking changes before `v1.0.0`

**Pull request merged**